salt/modules/network.py

```diff
@@ -1186,8 +1321,76 @@ def get_route(ip):
             'destination': ip,
             'gateway': m.group('gateway'),
             'interface': m.group('interface'),
-            'source': m.group('source')}
+            'source': m.group('source')
+        }
 
         return ret
+
+    if __grains__['kernel'] == 'SunOS':
+        # [root@nacl ~]# route -n get 172.16.10.123
+        #   route to: 172.16.10.123
+        #destination: 172.16.10.0
+        #       mask: 255.255.255.0
+        #  interface: net0
+        #      flags: <UP,DONE,KERNEL>
+        # recvpipe  sendpipe  ssthresh    rtt,ms rttvar,ms  hopcount      mtu     expire
+        #       0         0         0         0         0         0      1500         0
+        cmd = '/usr/sbin/route -n get {0}'.format(ip)
+        out = __salt__['cmd.run'](cmd, python_shell=True)
```

Comment:
This has the potential to allow for shell injections through carefully crafted arguments to this function. We can likely just change this to `python_shell=False` and it will be safe and still work fine.